packages/network_info_plus/network_info_plus/example/android/app/src/main/AndroidManifest.xml

```diff
@@ -2,6 +2,8 @@
     package="dev.fluttercommunity.plus.network_info_plus_example">
 
     <uses-permission android:name="android.permission.INTERNET" />
+    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
+    <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
```

Comment:
Added it, so it is easy to open Settings of `example` app and grant permission without introducing any dependency to handle permission in this app.